nginx/etc/confd/templates/supervisor/dhparam_regenerate.conf.tmpl

```diff
@@ -0,0 +1,5 @@
+[program:dhparam-regenerate]
+command = supervisorctl --user supervisor --password supervisor start dhparam-regenerate
```

Comment:
Will this be an infinite loop?

Reply:
no autorestart and startsecs set to 0, so it wont restart under any condition.

Reply:
oh oops I see what you mean

Reply:
I tested it somehow, maybe before I "corrected" a non-issue in `container --silent dhparam_regenerate`

Reply:
fixed